### PR TITLE
VAGOV-2855: Display 'Other VA locations' via Facility ID

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
Displays "Other VA Locations" on the locations page of a facility

I decided to add another react widget because of how often the designs change - I want the flexibility from the get-go since the design is in flux.

## Testing done
Locally with staging and dev data
Pre-req: May have to make sure there is data in the Pittsburgh Health Care system for "Other VA Locations" to populate `fieldOtherVaLocations` http://staging.cms.va.gov/node/318/edit?destination=/admin/content

You can fill in `nca_120` for the soldier lot that is in the mock ups

1. re-build site with either staging or dev data
2. browse to http://localhost:3001/pittsburgh-health-care/locations/
3. scroll all the way down to see the "Other VA Locations" section

## Screenshots
localhost_3001_pittsburgh-health-care_locations_ (2)

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-2855

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]
LGTM :+1: @ahay-agile6

File: src/site/layouts/health_care_region_locations_page.drupal.liquid [hunk 1]
```diff
@@ -19,6 +19,11 @@
                     <div class="cta-widget" data-widget-type="facility-locations-list" data-facilities="{{ mainFacilities.entities | widgetFacilitiesList | escape }}"></div>
                     <h2 id="community-clinic-locations">Community clinic locations</h2>
                     <div class="cta-widget" data-widget-type="facility-locations-list" data-facilities="{{ otherFacilities.entities | widgetFacilitiesList | escape }}"></div>
+
+                    {% if fieldOtherVaLocations != empty %}
+                        <h2 id="community-clinic-locations">Other nearby VA locations</h2>
+                        <div class="cta-widget" data-widget-type="other-facility-locations-list" data-facilities="{{ fieldOtherVaLocations }}"></div>
```
Comment (jbalboni) [inline src/site/layouts/health_care_region_locations_page.drupal.liquid]: Can we remove the cta-widget class from all these?

File: src/applications/static-pages/facilities/otherFacilityList.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      `[data-widget-type="other-facility-locations-list"]`,
+    ),
+  );
+
+  if (widgets.length) {
+    const {
+      default: OtherFacilityListWidget,
+    } = await import(/* webpackChunkName: "facility-locations-list" */ './OtherFacilityListWidget');
+
+    // since these widgets are on content pages, we don't want to focus on them
```
Comment (jbalboni) [inline src/applications/static-pages/facilities/otherFacilityList.js]: Can remove this comment

**Review by jbalboni — APPROVED** [review]
Couple minor changes, but LGTM otherwise

**Pull request merged**